apptools/io/tests/test_folder.py

```diff
@@ -108,77 +108,77 @@ def test_properties(self):
 
-        self.assertTrue(os.path.abspath(os.path.curdir) in f.absolute_path)
+        self.assertIn(os.path.abspath(os.path.curdir), f.absolute_path)
         self.assertEqual(len(f.children), 1)
         self.assertEqual(f.ext, '')
-        self.assertEqual(f.exists, True)
-        self.assertEqual(f.is_file, False)
-        self.assertEqual(f.is_folder, True)
-        self.assertEqual(f.is_package, True)
-        self.assertEqual(f.is_readonly, False)
+        self.assertTrue(f.exists)
+        self.assertFalse(f.is_file)
+        self.assertTrue(f.is_folder)
+        self.assertTrue(f.is_package)
+        self.assertFalse(f.is_readonly)
         self.assertEqual(f.mime_type, 'content/unknown')
         self.assertEqual(f.name, 'package')
         self.assertEqual(f.parent.path, 'data')
         self.assertEqual(f.path, 'data/package')
-        self.assertTrue(os.path.abspath(os.path.curdir) in f.url)
+        self.assertIn(os.path.abspath(os.path.curdir), f.url)
 
         return
 
     def test_copy(self):
         """ folder copy """
 
         f = File('data/sub')
-        self.assertEqual(f.exists, False)
+        self.assertFalse(f.exists)
 
         # Create the folder.
         f.create_folder()
-        self.assertEqual(f.exists, True)
+        self.assertTrue(f.exists)
         self.assertRaises(ValueError, f.create_folder)
 
         self.assertEqual(len(f.children), 0)
         self.assertEqual(f.ext, '')
-        self.assertEqual(f.is_file, False)
-        self.assertEqual(f.is_folder, True)
+        self.assertFalse(f.is_file)
+        self.assertTrue(f.is_folder)
         self.assertEqual(f.mime_type, 'content/unknown')
         self.assertEqual(f.name, 'sub')
         self.assertEqual(f.path, 'data/sub')
 
         # Copy the folder.
         g = File('data/copy')
-        self.assertEqual(g.exists, False)
+        self.assertFalse(g.exists)
 
         f.copy(g)
-        self.assertEqual(g.exists, True)
+        self.assertTrue(g.exists)
 
         self.assertEqual(len(g.children), 0)
         self.assertEqual(g.ext, '')
-        self.assertEqual(g.is_file, False)
-        self.assertEqual(g.is_folder, True)
+        self.assertFalse(g.is_file)
+        self.assertTrue(g.is_folder)
         self.assertEqual(g.mime_type, 'content/unknown')
         self.assertEqual(g.name, 'copy')
         self.assertEqual(g.path, 'data/copy')
 
         # Attempt to copy a non-existent folder (should do nothing).
         f = File('data/bogus')
-        self.assertEqual(f.exists, False)
+        self.assertFalse(f.exists)
 
         g = File('data/bogus_copy')
-        self.assertEqual(g.exists, False)
+        self.assertFalse(g.exists)
 
         f.copy(g)
-        self.assertEqual(g.exists, False)
+        self.assertFalse(g.exists)
 
         return
 
     def test_create_folder(self):
         """ folder creation """
 
         f = File('data/sub')
-        self.assertEqual(f.exists, False)
+        self.assertFalse(f.exists, False)
```

Comment:
The second argument should be removed here (or replaced with a suitable message, if you're feeling adventurous).
```suggestion
        self.assertFalse(f.exists)
```